dynd/__init__.py

```diff
@@ -19,3 +19,7 @@
     '__libdynd_version__', '__version__', '__libdynd_git_sha1__', '__git_sha1__',
     'annotate', 'test', 'load'
 ]
+
+from .nd.registry import propagate_all
```

Comment:
This breaks the nd/ndt separation since it makes dynd.ndt depend on having the nd subpackage available as well.